### PR TITLE
Try to fix PHP unit tests

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

Cause: https://github.com/WordPress/wordpress-develop/commit/8e498a831472cd0413ea0068480616fd5a0d5fe4

`var()` is now allowed, so we need to update the tests.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: phpunit/class-wp-theme-json-test.php [hunk 1]
```diff
@@ -567,109 +567,6 @@ function test_remove_insecure_properties_removes_unsafe_styles() {
...
-	function test_remove_insecure_properties_removes_unsafe_styles_sub_properties() {
-		$theme_json = new WP_Theme_JSON(
-			array(
-				'version' => WP_Theme_JSON::LATEST_SCHEMA,
-				'styles'  => array(
-					'spacing'  => array(
-						'padding' => array(
-							'top'    => '1px',
-							'right'  => '1px',
-							'bottom' => 'var(--unsafe-var-toplevel)',
```
Comment (ellatrix) [inline phpunit/class-wp-theme-json-test.php:579]: Maybe instead we should test for something else.

**Comment by oandregal** [comment]
@ellatrix got an alternative at https://github.com/WordPress/gutenberg/pull/31982

File: phpunit/class-wp-theme-json-test.php (hunk 1 above)
Comment (oandregal) [reply phpunit/class-wp-theme-json-test.php:579]: yup https://github.com/WordPress/gutenberg/pull/31982/files#diff-88c997ea8e8ba51f2fa201327e0f1faccd4e892c2fafe33249f72cfd068c72fcR579

File: phpunit/class-wp-theme-json-test.php [hunk 2]
```diff
@@ -586,7 +586,7 @@ function test_remove_insecure_properties_removes_unsafe_styles_sub_properties()
 								'padding' => array(
 									'top'    => '2px',
 									'right'  => '2px',
-									'bottom' => 'var(--unsafe-var-elements)',
```
Comment (ellatrix) [inline phpunit/class-wp-theme-json-test.php:589]: @nosolosw Btw, I noticed that these are not stripped.

**Pull request closed without merging**